data_sources/workzone.py

```diff
@@ -0,0 +1,153 @@
+import uuid
+from shapely.ops import linemerge
+import geopandas as gpd
+from shapely.geometry import shape, mapping
```

Comment:
something i wanted to mentioned when i submitted my review—these shapely methods are awesome. i'm really happy to see this library here. we have other ETLs out there that could benefit from these geometry ops 🙌